packages/amplify-cli/src/commands/status.ts

```diff
@@ -1,10 +1,22 @@
+import { constructCloudWatchEventComponent } from "amplify-category-function/src/provider-utils/awscloudformation/utils/cloudformationHelpers";
```

Comment:
where is this used?

Reply:
Its not currently used. It was left from an earlier attempt to save cloudwatch-logs from amplify-push into a logfile and to surface them as error in status summary. removing this for now.